README.md

```diff
@@ -15,11 +15,14 @@ Ready to use blocks to configure popular tools like *Babel*, *PostCSS*, *Sass*,
 ## Table of Contents
 
 - [Installation](#installation)
-- [Usage](#usage)
-- [Available webpack blocks](#available-webpack-blocks)
-- [Design principles](#design-principles)
+- [Example](#example)
+- [More Examples](#more-example)
+- [Custom Blocks](#custom-blocks)
+- [Available Webpack Blocks](#available-webpack-blocks)
+- [Third-party Blocks](#third-party-blocks)
+- [Design Principles](#design-principles)
 - [FAQ](#faq)
-- [Like what you see?](#like-what-you-see)
+- [Like What You See?](#like-what-you-see)
```

Comment:
No, it’s a heading, so it should be capitalized as all other headings. Personally I never use title case but I see that @andywer is using it in other documents.